File: Assignment(G2)/A6_108502516/A6_108502516.java
/*
 * Assignment 6
 * Course: CE1004-A
 * Name: ¡¬À»È∫
 * Student ID: 108502516
 */
package javatest;
import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;

public class A6_108502516 extends Application {
    @Override
    public void start(Stage primaryStage) throws Exception{
        Parent root = FXMLLoader.load(getClass().getResource("A6_108502516.fxml"));
        primaryStage.setTitle("Hello World");
        primaryStage.setScene(new Scene(root, 600, 400));
        primaryStage.show();
    }
    public static void main(String[] args) {
        launch(args);
    }
}

File: Assignment/A6/A6-108502516.cpp
/*
*   Assignment 6
*   Name: ¡¬À»È∫
*   Student Number: 108502516
*   Course: 2019-CE1003-A
*/
#include<iostream>
#include<string>

using namespace std;

int main()
{
    int n;
    cin >> n;
    for(int a=0;a<n;a++)
    {
        char eng;int num;
        string s1;
        cin >> s1;
        for(int b=0;b<s1.size();b++)
        {
            if(s1[b]<='Z'&&s1[b]>='A')
                eng=s1[b];
            if(s1[b]<='9'&&s1[b]>='0')
            {

                if(s1[b+1]<='9'&&s1[b+1]>='0')
                {
                    num=(s1[b]-'0')*10+(s1[b+1]-'0');
                    b=b+1;
                    while(s1[b+1]<='9'&&s1[b+1]>='0')
                    {
                        num=num*10+(s1[b+1]-'0');
                        b++;
                    }
                }
                else
                    num=s1[b]-'0';
                for(int c=0;c<num;c++)
                    cout << eng;
            }
        }
        cout << endl;
    }
}
